libs/coin-modules/coin-tezos/src/logic/listOperations.ts

```diff
@@ -48,7 +58,12 @@ function convertOperation(
     value: BigInt(amount),
     // storageFee for transaction is always present
     fee: BigInt(storageFee ?? 0),
-    blockHeight: 0, // operation.block is a string
+    blockHeight: block.level,
+    block: {
+      hash: block.hash,
+      height: block.level,
+      time: new Date(block.timestamp),
+    },
```

Comment:
```suggestion
    block: {
      hash: operation.block,
      height: operation.level,
      time: new Date(operation.timestamp),
    },
``` 


<img width="949" alt="Shottr 2024-12-23 16 14 57" src="https://github.com/user-attachments/assets/de1f90e8-a347-49f7-b7d7-ddc1e37e620d" />

As per the doc and @sprohaszka-ledger, we can indeed rely only on the data found in the operation object